### ISSUE TITLE
how to position upside down model

### ISSUE THREAD

**Issue by Lutr1z** [google/model-viewer#3370]
### Description

Hello, I am using model-viewer to display a glb model generated by colmap. The model generated is upside down, so it's also displayed upside down in the model-viewer. I enabled camera-control, but seems it has a maximum orbit angle limitation, so i can not get a correct position model. 
Is there any method to set the original position or rotation of model on load(i tried three.js and it has such function), or how can i change the max orbit angle of camera?

### Version

- model-viewer: 1.11.1

**Comment by elalish** [comment]
Yes! https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-orientation
